test/CMakeLists.txt

```diff
@@ -58,3 +58,9 @@ add_lit_testsuite(test-dsa "Regression test for DSA"
   test_dir=${CMAKE_CURRENT_BINARY_DIR}
   DEPENDS seahorn
   )
+
+install (DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/simple DESTINATION test)
```

Comment:
please change destination to `share/searhon/test` to be consistent with typical location of files on a linux system